desktop-widgets/preferences/preferences_cloud.cpp

```diff
@@ -36,6 +37,7 @@ void PreferencesCloud::syncSettings()
 	QString email = ui->cloud_storage_email->text().toLower();
 	QString password = ui->cloud_storage_password->text();
 	QString newpassword = ui->cloud_storage_new_passwd->text();
+	QString emailpasswordformatwarning = "Change ignored. Cloud storage email and new password can only consist of letters, numbers, and '.', '-', '_', and '+'.";
```

Comment:
Attention: This breaks translation.
You most likely want
```
        QString emailpasswordformatwarning = tr("Change ignored. Cloud storage email and new password can only consist of letters, numbers, and '.', '-', '_', and '+'.");
```

This does the translation every time, even if the error isn't raised. Personally, I don't think that this is a problem.

Reply:
Good catch, thanks.